includes/admin/class-give-welcome.php

```diff
@@ -175,11 +175,11 @@ public function get_welcome_header() {
 
 			<div class="give-welcome-header-inner">
 
-				<h1 class="give-welcome-h1"><?php esc_html_e( $title ); ?></h1>
+				<h1 class="give-welcome-h1"><?php esc_html_e( $title, 'give' ); ?></h1>
```

Comment:
Translations don't work with dynamic data.

Reply:
Should just be `echo esc_html( $title )`.

Reply:
@bradp Thank you for catching that. Textdomain added by linter.